packages/ai-api/src/utils/deployment-resolver.test.ts

```diff
@@ -114,6 +117,44 @@ describe('deployment resolver', () => {
 
     expect(id).toEqual('5');
   });
+
+  describe('get all deployments', () => {
```

Comment:
Why did we randomly introduce a new test here?

Reply:
Long story. Because the first task of the BLI was to improve this function. And I first add `ErrorWithCause` with tests only to this one. And then I realize that it can be extracted to a lower level execute request. But since the test is harmless, I left it their.